### DISCUSSION TITLE
Setting option without a config file

### DISCUSSION THREAD

**Discussion by IzhakJakov** [zellij-org/zellij#3487] [General]
In the [documentation for the options](https://zellij.dev/documentation/options) it states that:

> Configuration options can be set directly at the root of the [configuration file](https://zellij.dev/documentation/configuration)

Is that the only way to set the options?

I would like to be able to try out the options before adding them to my config is there a way to do it?

**Comment by imsnif** [comment]
Sure - you can do it through the command line. Check out `zellij options --help` for more info.

**Reply by IzhakJakov** [reply]
Thanks for the quick reply.
